pyproject.toml

```diff
@@ -124,22 +124,51 @@ write_to = "plasmapy/_version.py"
 
 [tool.pytest.ini_options]
 testpaths = ['plasmapy']
-norecursedirs = ['build', 'docs', 'plasmapy[\/]_dev']
 xfail_strict = true
+markers = [
+    "slow: marks tests as slow (deselect with '-m \"not slow\"')",
+]
 doctest_optionflags = """
 NORMALIZE_WHITESPACE
 ELLIPSIS
-NUMBER"""
+NUMBER
+IGNORE_EXCEPTION_DETAIL"""
 filterwarnings = [
   "ignore:.*deprecated in traitlets 4.2.*:DeprecationWarning",
   "ignore:.*MultiIndex.*:DeprecationWarning",  # https://github.com/PlasmaPy/PlasmaPy/issues/2319
 ]
+norecursedirs = [
+  '*egg-info*',
+  '.*cache*',
+  '.*env',
+  '.[tn]ox',
+  '.cov*',
+  '.git*',
+  '.hypothesis',
+  '.idea',
+  '.jupyter',
+  '.vscode',
+  '__pycache__',
+  '_dev',
+  'binder',
+  'build',
+  'changelog',
+  'dist',
+  'docs',
+  'licenses',
+  'node_modules',
+  'venv',
+]
+
 
 [tool.coverage.report]
 exclude_lines = [
+  "if TYPE_CHECKING",
+  "if typing.TYPE_CHECKING",
```

Comment:
These blocks will never be run during tests since they are only get run when mypy is doing its type checking thing.